fix: Add readme for confidence flag api (#53)
spotify/confidence-openfeature-provider-python: README.md

### Creating and using the flag provider

> Below is an example for how to create a Confidence client, and then resolve
> a flag with a boolean attribute. The provider is configured with an api key and a region, which will determine
> where it will send the resolving requests. 

Reviewer: ```suggestion
Below is an example for how to initialize the Confidence SDK, and then resolve
a flag with a boolean attribute. The SDK is configured with an api key, which will authorize the resolving requests. 
```

```python

from confidence.confidence import Confidence
from confidence.confidence import Region

> confidence = Confidence("API_KEY", region=Region.EU)

Reviewer: ```suggestion
confidence = Confidence("API_KEY")
```
